.github/workflows/base-ci-goreleaser.yaml

```diff
@@ -57,7 +57,8 @@ jobs:
         with:
           distribution: goreleaser-pro
           version: latest
-          args: --snapshot --clean --skip-sign --skip-sbom --timeout 2h --split --id ${{ inputs.distribution }}
+          workdir: distributions/${{ inputs.distribution }}
+          args: --snapshot --clean --skip-sign --skip-sbom --timeout 2h --split
```

Comment:
@jpkrohling in this situation I feel ok with the skips.  That particular log is for linux_amd64, and it does say

```
    • building docker image                          image=otel/opentelemetry-collector:0.95.0-SNAPSHOT-a5b01f6-amd64
```

which indicates that it built the linux amd64 image.  All other images are skipped since they dont match this matrix instance's os and architecture.

Can you confirm that the linux amd64 image built?

Reply:
Comparing with the old job, you can see it did the skips also bc it also used a matrix: https://github.com/open-telemetry/opentelemetry-collector-releases/actions/runs/7832341341/job/21370820540